velox/type/TimestampConversion.h

```diff
@@ -107,6 +107,24 @@ Status daysSinceEpochFromWeekDate(
     int32_t dayOfWeek,
     int64_t& out);
 
+/// Computes the (signed) number of days since unix epoch (1970-01-01).
+/// Returns error status if the date is invalid. We treat days of the previous
+/// or next months as a part of the specified WEEK_OF_MONTH. For example, if
+/// weekOfMonth is 5 but the current month only has 4 weeks (such as February),
+/// the first week of March will be considered as the 5th week of February.
+/// @param year Year, can be negative e.g: 1996, -2000
+/// @param month Month of year, A value in [1, 12] range. For example, 1 is Jan,
+/// 7 is Jul.
+/// @param weekOfMonth Week of the month. A value in [1, 5] range. For example,
+/// 1 is 1st week, 3 is 3rd week.
+/// @param dayOfWeek Day number of week. A value in [1, 7] range. For example, 1
+/// is Monday, 7 is Sunday.
+Expected<int64_t> daysSinceEpochFromWeekOfMonthDate(
+    int32_t year,
```

Comment:
The implementation imposes limits on the 'year' value. Would you document these?

Reply:
Thanks. Done.